tests/emformer/test_modeling_emformer.py

```diff
@@ -0,0 +1,1556 @@
+# coding=utf-8
```

Comment:
Note that we should rebase this PR to the correct format -> tests are now under `tests/models/emformer`